pkg/utils/annotations.go

```diff
@@ -71,62 +70,89 @@ func AppContainerPostStartInjectEnabled(infos map[string]string) (match bool) {
 
 // ---- Utils functions to decide serverless platform ----
 const (
-	PlatformDefault      = "Default"
-	PlatformUnprivileged = "Unprivileged"
+	PlatformDefault      = "default"
```

Comment:
I suggest  Using `ServerlessPlatformDefault` and `ServerlessPlatformUnprivileged ` as keyName.

Reply:
Done.